components/pipeline-service/development/main-pipeline-service-configuration.yaml

```diff
@@ -2101,7 +2101,7 @@ metadata:
   namespace: openshift-marketplace
 spec:
   displayName: custom-operators
-  image: quay.io/openshift-pipeline/pipelines-index-4.15@sha256:bf032558eeb696ed351cb9c00503f6d415e4764639ef23295abd51bad566a966
+  image: quay.io/openshift-pipeline/pipelines-index-4.15:next
```

Comment:
I prefer to not use a floating tag since it makes it harder to understand why errors happen (we don't know what the content it points to changes).